docs/versioned/admin/admin-overview.md

```diff
@@ -7,40 +7,86 @@ function: explanation
+    Eventing --> event-impl["Implements"]
+    event-impl --> event-kafka & event-rabbitmq & event-nats
+    dev-acts@{ shape: text}
+    admin-acts@{ shape: text}
+    net-impl@{ shape: text}
+    event-impl@{ shape: text}
+    style Serving fill:#D5D5D5,color:#000000
+    style Eventing fill:#F0DBDB,color:#000000
+    style net-istio fill:#D5D5D5,color:#000000
+    style net-contour fill:#D5D5D5,color:#000000
+    style net-gateway-api fill:#D5D5D5,color:#000000
+    style event-kafka fill:#F0DBDB
+    style event-rabbitmq fill:#F0DBDB
+    style event-nats fill:#F0DBDB
+    style Dev fill:#EFB769,color:#000000
+    style dev-acts fill:transparent
+    style Admin fill:#94C6C1,color:#000000
+    style admin-acts fill:transparent
+    style net-impl fill:transparent
+    style event-impl fill:transparent
+```
 
 As a cluster administrator, your responsibilities include managing the Kubernetes environment, installing cluster-wide components, and enabling developers to deploy applications on the cluster. Knative aims to simplify developer tasks, while aligning with existing management tools and processes.
 
 Knative includes a plugin system to integrate with existing infrastructure in the cluster, enabling Knative resources such as Routes and Brokers to be implemented using one of multiple underlying suppliers. For example, a Knative Eventing app can deliver events to a Broker that triggers a function based on the received event. In a testing cluster, the delivery might use an in-memory option, while a staging or production environment might use a cloud-provided Kafka service.
 
 Of particular interest to cluster administrators is that Knative supports customizable _default values_ on the parameters defined in resource YAML files. These configurations reduce the amount of environment configuration tasks developers needs to consider.
 
-## Knative installations
+## Installation decisions
 
 See the [Installation roadmap](../install/README.md#installation-roadmap) for prerequisites and installation steps. Your first installation decision is whether to use a YAML-based installation or use the Knative Operator. The Knative Operator is a custom controller that extends the Kubernetes API to install Knative components. If you just need to get acquainted with Knative at this time, you can install the [quickstart](../getting-started/quickstart-install.md).
 
-## Configuring Knative
-
-Knative uses Kubernetes YAML manifests to define and configure system components. These manifests include core resources, custom resource definitions (CRDs), and extensibility features. As with Kubernetes, these configuration resources are declarative and can be managed using the `kubectl` CLI tool or with continuous delivery tools.
+Knative installations are not permanent and you can install clusters differently depending on the situation. Although switching services and applications in real time may be possible, a best practice is to start with a new cluster.
```

Comment:
I'm not sure what "are not permanent" is supposed to imply.  Maybe "The method you use to install Knative is not permanent ..."?

I think the second sentence also slightly missed my point -- "Although transitioning between installation methods on one cluster is possible, new installations on separate clusters is the better-tested and officially supported approach.